src/main/java/net/comments/ft/Test3.java

```diff
@@ -0,0 +1,43 @@
+package net.comments.ft;
+
+import net.comments.objects.BCommentPage;
+import net.comments.objects.CommentPage;
+import net.comments.objects.CommentTable;
+import org.hamcrest.MatcherAssert;
+import org.testng.annotations.Test;
+
+public class Test3 extends BaseDriver {
+    private final CommentPage page;
+
+    Test3() {
+        this.page = new BCommentPage(this.driver);
+    }
+
+    @Test
+    public void test() {
+        page.open();
+        int commentNumber = 1;
+        page.getCurrentComments().selectComment(commentNumber);
+        String oldState = page.getCurrentComments().getCommentState(commentNumber);
+        page.getCommentAction().edit();
+        String newNumber = "45";
+        page.getCommentModifier().fillCommentNumber(newNumber);
+        String newText = "Edited Comment Text " + commentNumber;
+        page.getCommentModifier().fillCommentText(newText);
+        String newState = null;
```

Comment:
This variable is not used in the code below. I suppose it has to be removed.

Reply:
@extsoft  I add method with using of "newState" variable